db/migrate/20151203230940_add_date_type_to_content_view_erratum_filter_rules.rb

```diff
@@ -0,0 +1,5 @@
+class AddDateTypeToContentViewErratumFilterRules < ActiveRecord::Migration
+  def change
+    add_column :katello_content_view_erratum_filter_rules, :date_type, :string, :default => "issued", :null => false
```

Comment:
For example -- https://github.com/Katello/katello/blob/master/engines/bastion_katello/app/assets/javascripts/bastion_katello/errata/views/errata-table-full.html#L43